connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3393,6 +3421,81 @@ object functions {
    */
   def upper(e: Column): Column = Column.fn("upper", e)
 
+  /**
+   * Converts the input `e` to a binary value based on the supplied `f`. The `f` can be a
+   * case-insensitive string literal of "hex", "utf-8", "utf8", or "base64". By default, the
+   * binary format for conversion is "hex" if `fmt` is omitted. The function returns NULL if at
+   * least one of the input parameters is NULL.
+   *
+   * @group string_funcs
+   * @since 3.5.0
+   */
+  def to_binary(e: Column, f: Column): Column = Column.fn("to_binary", e, f)
+
+  /**
+   * Converts the input `e` to a binary value based on the format "hex". The function returns NULL
+   * if at least one of the input parameters is NULL.
+   *
+   * @group string_funcs
+   * @since 3.5.0
+   */
+  def to_binary(e: Column): Column = Column.fn("to_binary", e)
+
+  // format: off
```

Comment:
I think we should not use `// format: off`

Reply:
Okay, let me reorganize this comment again.

Reply:
- Before:
<img width="777" alt="image" src="https://github.com/apache/spark/assets/15246973/e9ec5f23-ace6-4f05-9564-33932de10947">

- After remove // format: off ...
<img width="774" alt="image" src="https://github.com/apache/spark/assets/15246973/4331663e-2e40-4d09-9efa-0a44eeaf786e">

- Because the `connect` module has strict scala style detection. We have to format it, but then the readable of comment is poor. Should we do this - `remove // format: off`?
https://github.com/apache/spark/blob/c30b411e412a76bf85d6e438e6b0c39014476a0c/dev/lint-scala#L23-L45

Reply:
I guess you can refer to https://github.com/apache/spark/blob/f1cca85f51322bac3a75346fa2d01e4133f6e31c/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L3328-L3345 ?

Reply:
Done